### ISSUE TITLE
Typescript compilation error related to Trusted Types

### ISSUE THREAD

**Issue by pshihn** [lit/lit#1780]
# Description

After updating from `"lit": "^2.0.0-pre.2"` to `"lit": "^2.0.0-rc.1"` my typescript project compilation fails with the following errors:

```
node_modules/lit-html/lit-html.d.ts:6:23 - error TS2688: Cannot find type definition file for 'trusted-types'.
 /// <reference types="trusted-types" />
```

```
node_modules/lit-html/lit-html.d.ts:66:8 - error TS2304: Cannot find name 'TrustedHTML'.
h: TrustedHTML;
```

**Comment by pshihn** [comment]
Seems like default typescript DOM lib does not have TrustedTypes in them.

**Comment by rictic** [comment]
Thanks for the bug report, we should do something about this.

Two workarounds. You can install the TrustedTypes typings by running:

```
npm install "@types/trusted-types"
```

Or you can skip doing exhaustive type checking for dependencies by adding:

```
"skipLibCheck": true,
```

to the `compilerOptions` section of your tsconfig.json file

**Comment by rictic** [comment]
Fixed in `lit-html@2.0.0-rc.2`

Thank you for trying it out so quickly and filing such a clear and helpful bug!
